kotlin-css/src/commonMain/kotlin/kotlinx/css/StyleEnums.kt

```diff
@@ -38,6 +41,16 @@ enum class Appearance {
     override fun toString() = name.hyphenize()
 }
 
+enum class Isolation {
+    initial, inherit, revert, unset,
+
+    auto, isolate,
+
+    ;
+
+    override fun toString() = name.hyphenize()
```

Comment:
Not needed as none of the values require hyphenization.

Reply:
Fixed